docs/plugin-vidarr.md

```diff
@@ -22,7 +20,14 @@ If a number, in seconds, is given, then the action must have been
 window is exceeded, they will be run as dry-run.
 
 The Vidarr plugin provides an input format: `vidarr_analysis` which provides the analysis
-provenance data. 
+provenance data. To use this input format add the fields to the configuration file as follows:
+
+    {
+      "defaultMaxSubmissionDelay": null,
+      "url": "http://vidarr:8000",
+      "analysisTypes": ["FILE"],
+      "versionTypes": []
+    }
```

Comment:
Again, this could be mistaken for needing to be two parallel objects in the config file. 

```suggestion
provenance data. To use this input format add the fields to the configuration file as follows:

      "analysisTypes": ["FILE"],
      "versionTypes": []
```